source/development/packaging/generate-dashboard-package.rst

```diff
@@ -20,15 +20,22 @@ Build manually
 
 Requirements:
 
--  Docker
+-  Docker: see `Docker installation guide <https://docs.docker.com/engine/install/>`_
+-  zip: see `zip installation guide <https://www.tecmint.com/install-zip-and-unzip-in-linux/>`_
```

Comment:
**suggestion**: it would be more convenient to provide an official reference as possible, else we could add instructions in the documentation or only list the dependency.